src/components/related-links/_related-links.scss

```
@@ -0,0 +1,4 @@
+.related-links {
+  padding: 15px 0;
```

@will-madebymoon might hurt you if you don't stick to his multiple of 9's spacing rule 👀

```suggestion
  padding: 1rem 0;
```